src/core/ecetFactory.h.in

```diff
@@ -1,3 +1,5 @@
+#pragma once
+
 /*******************************************************************************
  * Copyright (c) 2018 fortiss GmbH
```

Comment:
I missed that the last time, please change to 2018, 2024 fortiss GmbH, Jose Cabral

Reply:
done